File: 4-spacejam/spacejam.py
"""
Given a string, remove all spaces from the string, insert two spaces 
between every character, convert all alphabetical letters to uppercase, and return the result.

Non-alphabetical characters should remain unchanged (except for spaces).
"""

def space_jam(s):
    """
    Generates a double space between each character
    Args:
        s(string):The input string to be processed
    Returns:
        spacejam(string): The processed string with double spaces between each character
    """
    removed_space = s.replace(" ", "")
    spacejam_s = "  ".join(removed_space.upper())
    return spacejam_s

#Test stage
print(space_jam("spacejam"))

File: 0004-spacejam/spacejam.js
function spaceJam(s) {

  let removed_space = s.replace(/\s+/g, "");
  let spacejam_s = removed_space.split('').join('  ').toUpperCase();
  
  return spacejam_s;
}

// Test Stage
console.log(spaceJam("hello world")); 